core/src/main/java/cucumber/runtime/formatter/PrettyFormatter.java

```diff
@@ -150,8 +151,17 @@ private void handleTestStepFinished(TestStepFinished event) {
     }
 
     private void handleWrite(WriteEvent event) {
-        out.println(event.text);
-    }
+        out.println();
+        try(BufferedReader lines = new BufferedReader(new StringReader(event.text))) {
+			String line;
+			while ((line = lines.readLine()) != null) {
```

Comment:
I don't mean to keep going on about this but please use spaces for indentation rather then tabs.

We've documented the settings to configure your IDE here:

https://github.com/cucumber/cucumber-jvm/blob/master/CONTRIBUTING.md#contributing

And we also provide a .editorconfig file that most IDEs will pick up automatically.

https://github.com/cucumber/cucumber-jvm/blob/master/.editorconfig